src/Symfony/Component/Workflow/Definition.php

```diff
@@ -28,9 +28,10 @@
     private $metadataStore;
 
     /**
-     * @param string[]     $places
-     * @param Transition[] $transitions
-     * @param string|null  $initialPlace
+     * @param string[]                    $places
+     * @param Transition[]                $transitions
+     * @param string|null                 $initialPlace
+     * @param MetadataStoreInterface|null $metadataStore
```

Comment:
We prefer doing partial docblocks, keeping them only when they provide something over the signature:
instead of adding this, I'd suggest to remove the one for initialPlace